testing/test_collection.py

```diff
@@ -1178,6 +1179,15 @@ def test_nodeid(request):
     assert result.ret == 0
 
 
+def test_collect_symlink_dir(pytester: Pytester) -> None:
+    """A symlinked directory is collected."""
+    dir = pytester.mkdir("dir")
+    dir.joinpath("test_it.py").write_text("def test_it(): pass", "utf-8")
+    pytester.path.joinpath("symlink_dir").symlink_to(dir)
```

Comment:
Hmm, should have used `symlink_or_skip` here, and absent that, should have used `symlink_to(dir, target_is_directory=True)`. I wonder how it passed windows CI without it. Will fix it anyway.